tests/compile-fail/panic/rustc_allocator_nounwind.rs

```diff
@@ -0,0 +1,10 @@
+#![feature(rustc_attrs, c_unwind)]
+
+#[rustc_allocator_nounwind]
+extern "C-unwind" fn nounwind() {
+    panic!();
+}
+
+fn main() {
+    nounwind(); //~ ERROR unwinding past a stack frame that does not allow unwinding
```

Comment:
You already tested this attribute above in `exported_symbol_bad_unwind2.rs`, why test it again here?

Reply:
The function is not called via an `extern` block in this test, so it's slightly different.